README.md

```diff
@@ -30,89 +81,164 @@ $response = $fraudService->validateRequest($request);
+
+Once you have your random session ID, toss it into a cookie (or custom header, however you want to communicate it) and use it for both the front-end tracker and the back-end fraud requests.
+
+### Instantiation
+
+Instantiation of a driver is incredibly straightforward.
+
+You can create them yourself, passing all configs necessary as the first and only constructor argument.
+
+```php
+<?php
+
+use Omnifraud\Kount\KountService;
+
+/** @var \Omnifraud\Contracts\ServiceInterface $fraudService */
+$fraudService = new KountService([
+    'apiKey' => 'XXX',
+    'merchantId' => '123456',
+]);
 ```
 
-Each package already requires [`omnifraud/common`](https://github.com/lxrco/omnifraud-common),
-so you don't need to require it.
+Or, use the static `create()` method offered by `Omnifraud\Omnifraud`. The first parameter is the **alias** of the driver (detailed in the [table above](#fraud-services--drivers)), and the second is the same configuration array you would normally supply.
 
-You can also install ALL supported services:
+```php
+<?php
 
-```bash
-composer require omnifraud/omnifraud
+use Omnifraud\Omnifraud;
+
+/** @var \Omnifraud\Contracts\ServiceInterface $fraudService */
+$fraudService = Omnifraud::create('Kount', [
+    'apiKey' => 'XXX',
+    'merchantId' => '123456',
+]);
 ```
 
+As every driver is an implementation of `Omnifraud\Contracts\ServiceInterface`, you should be careful to ensure that you typehint this contract instead of any concrete implementations. Doing anything else would defeat the purpose of this library.
 
-## Fraud services
+### Front-end Implementation
 
-All drivers must implement [ServiceInterface](https://github.com/lxrco/omnifraud-common/blob/master/src/Contracts/ServiceInterface.php).
+All services expose a `trackingCode(string $pageType, string $sessionId)` method which returns a stringified snippet of JavaScript. You can call this method to insert the code necessary to instrument the front-end tracking of your fraud service. The two required parameters are a constant to specify the type of page we're inserting the snippet into, and the clients [session ID](#the-session-id).
 
-The following services are officially supported right now:
+Be sure to pass the appropriate constant as some services will differentiate between the two types of pages. It can be one of these two values:
+
+* `ServiceInterface::PAGE_ALL`
+* `ServiceInterface::PAGE_CHECKOUT`
 
-Service | Composer Package | Maintainer
---- | --- | ---
-[Kount](https://github.com/lxrco/omnifraud-kount) | omnifraud/kount | [LXRandCo](https://github.com/lxrco)
-[Signifyd](https://github.com/lxrco/omnifraud-signifyd) | omnifraud/signifyd | [LXRandCo](https://github.com/lxrco)
+Quite simply, the return string is an IIFE including the session ID. To help clarify how this works let's take a peak at the `SignifydService`. Its `trackingCode()` method will return the following JavaScript snippet:
 
+```javascript
```

Comment:
This is nice, but it would be nice to have an example of how to use the code (I know its simple, but just to make is super clear), probably in pure php like `<?=$service->trackingCode(ServiceInterface::PAGE_CHECKOUT);?>`

Reply:
:+1: